aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -140,16 +140,22 @@ module std::features {
         is_enabled(DELEGATION_POOLS)
     }
 
-    /// Whether the basic operations over some BLS12381 structures are enabled in the algebra module.
+    /// Whether generic algebra basic operation support in `algebra.move` are enabled.
     ///
-    /// Basic operations include element (de)serialization, field/group arithmetic, hash-to-structure, casting, etc.
-    /// BLS12381 structures controlled by this flag includes `Fq12`, `G1`, `G2`, `Gt`, `Fr`.
+    /// Lifetime: transient
+    const GENERIC_ALGEBRAIC_STRUCTURES_BASIC_OPERATIONS: u64 = 12;
```

Comment:
We probably need 2 flags.
`GENERIC_ALGEBRAIC_STRUCTURES_BASIC_OPERATIONS` is for gating on the Move side before calling into rust backend.
`BLS12_381_STRUCTURES` is needed inside rust backend to control the native behavior.

Reply:
I don't think we need `BLS12_381_STRUCTURES`.

If GENERIC_ALGEBRAIC_STRUCTURES_BASIC_OPERATIONS is **on** but BLS12_381_STRUCTURES is **off**, you call into a native that does nothing & aborts.

But there is no point to it.

I would just use one flag (the previously-defined BLS12381_BASIC_OPERATIONS) to feature gate the calls to the natives inside Move.

Reply:
One flag will certainly work for the purpose of this change, but having generic functions controlled by a flag with `BLS12381` in its name is a bit weird...

Reply:
> If GENERIC_ALGEBRAIC_STRUCTURES_BASIC_OPERATIONS is on but BLS12_381_STRUCTURES is off

Not likely, but in case we need to retire BLS12381 but keep the generic operations for other groups, having orthogonal flags makes it easy.

Reply:
Hm, I see what you're saying: that might help us turn the BLS12-381 functionality off fast via a governance proposal, as opposed to a hotfix.

Let's keep it then.

And let's abbreviate `GENERIC_ALGEBRAIC_STRUCTURES_BASIC_OPERATIONS` to something shorter? _Maybe_ `CRYPTOGRAPHY_ALGEBRA_NATIVES`?

Reply:
I actually need suggestions for this: does it sound good to use `CRYPTOGRAPHY_ALGEBRA_NATIVES` for this initial list of operations, and a new flag like `CRYPTOGRAPHY_ALGEBRA_NATIVES_VOL_2` when later we add new operations?

Reply:
Fixed.

Reply:
No `CRYPTOGRAPHY_ALGEBRA_NATIVES_VOL_2` please. It's non-descriptive. 

Let's use 'CRYPTOGRAPHY_ALGEBRA_NATIVES' for the 1st and 'BLS12_381_STRUCTURES' for the 2nd, like you have it now (or '_ALGEBRA', if you want).